regolith/helpers/a_proposalhelper.py

```diff
@@ -0,0 +1,197 @@
+"""Helper for adding a proposal to the proposals.yml collection.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+    fuzzy_retrieval
+)
+
+TARGET_COLL = "proposals"
+
+def subparser(subpi):
+    # Do not delete --database arg
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("name", help="A short but unique name for the proposal",
+                       default=None
+                       )
+    subpi.add_argument("amount", help="value of award",
+                       default=None
+                       )
+    subpi.add_argument("authors", nargs="+",
+                       help="Other investigator names"
+                       )
+    subpi.add_argument("date", help="The begin date for the proposed grant "
+                                          "in format YYYY-MM-DD."
+                       )
+    subpi.add_argument("end_date", help="The end date for the proposed grant "
+                                        "in format YYYY-MM-DD."
+                       )
+    subpi.add_argument("due_date", help="The due date for the proposal in format YYYY-MM-DD."
+                       )
+    subpi.add_argument("duration", help="Duration of proposal in years"
```

Comment:
first, I think that this should be in months.  Are you sure it is in years?   Second, if we know the begin and end-date this can be computed. Maybe request begin_date and right after, request duration, then compute the end-date internally.

Reply:
done